### DISCUSSION TITLE
Hi-Res Web Radio URL

### DISCUSSION THREAD

**Discussion by roadsterm** [rern/rAudio#1585] [General]
These web radio stations broadcast lossless FLAC stream from 44.1 kHz  up to192 kHz.

I have added these web radios to my  rAudio (20240127) running in Pi4B, it could play most of them flawlessly except SuperStereo 1 - 7, rAudio can't play continuously. I've to play SuperStereo 1 CD  to 7 CD instead.

For more information please visit this web.
https://www.hiresaudio.online/cd-quality-internet-radio/

Enjoys

Hi-Res Web Radio

Mother Earth Radio
https://motherearth.streamserver24.com/listen/motherearth/motherearth

Mother Earth Klassik
http://motherearth.streamserver24.com:18910/motherearth.klassik

Mother Earth Instrumental
http://motherearth.streamserver24.com:18920/motherearth.instrumental

Mother Earth Jazz
https://motherearth.streamserver24.com/listen/motherearth_jazz/motherearth.jazz

SuperStereo 1
http://icecast.centaury.cl:7570/SuperStereoHiRes1

SuperStereo 1 CD
http://icecast.centaury.cl:7560/SuperStereoNoData1

SuperStereo 1+
http://ingest-mia.radiomast.io/8a760c25-fb95-4fcb-9c0e-ca0f269a7360

SuperStereo 2
http://icecast.centaury.cl:7570/SuperStereoHiRes2

SuperStereo 2 CD
http://icecast.centaury.cl:7560/SuperStereoNoData2

SuperStereo 3
http://icecast.centaury.cl:7570/SuperStereoHiRes3

SuperStereo 3 CD
http://icecast.centaury.cl:7560/SuperStereoNoData3

SuperStereo 4
http://icecast.centaury.cl:7570/SuperStereoHiRes4

SuperStereo 4 CD
http://icecast.centaury.cl:7560/SuperStereoNoData4

SuperStereo 5
http://icecast.centaury.cl:7570/SuperStereoHiRes5

SuperStereo 5 CD
http://icecast.centaury.cl:7560/SuperStereoNoData5

SuperStereo 6
http://icecast.centaury.cl:7570/SuperStereoHiRes6

SuperStereo 6 CD
http://icecast.centaury.cl:7560/SuperStereoNoData6

SuperStereo 7
http://icecast.centaury.cl:7570/SuperStereoHiRes7

SuperStereo 7 CD
http://icecast.centaury.cl:7560/SuperStereoNoData7

ČRo Vltava
http://amp.cesnet.cz:8000/cro3.flac

ČRo D-dur
http://amp.cesnet.cz:8000/cro-d-dur.flac

ČRo Jazz
http://amp.cesnet.cz:8000/cro-jazz.flac

Radio Calico
http://radio3.radio-calico.com:8080/calico

**Comment by cmh714** [comment]
I only get that with Hi-Res over 750kHz but these channels are more normal at 96k...interesting.

**Reply by roadsterm** [reply]
Thanks for your comment.

Let me explain more about " can't play continuously".

When my rAudio play SuperStereo 1 -7 (96kHz), it plays smoothly  until  second track  start for a few seconds, there is no sound and cover is not changed. I've to click stop and play button then second track is played and cover art is changed.

This problem dose not occur when it plays other channels even 192 kHz with cover art  like SuperStereo 1+.

Did you find any problem like this?

**Reply by cmh714** [reply]
Only with 750+. Everything from 192 and below works fine here so far. But in testing those stations you provided, yes they seem to stop between songs. Maybe something odd in the signal they send???? not sure.

**Reply by roadsterm** [reply]
Thanks for your very quick response.

**Reply by Raitsa** [reply]
I have experienced this same "stop playing" behaviour with a few other FLAC radio streams as well.

I have tested and/or added to my favourites most of these:
https://www.hiresaudio.online/cd-quality-internet-radio/

**Comment by Raitsa** [comment]
@Club80 
Club 80 all streams working nicely, good stuff, thanks.
Artist/song infos (metadata) not showing though.

Screenshot taken from radiolise.com 
Your streams can also be found with radio-browser.info

![2024-12-30_17-17](https://github.com/user-attachments/assets/f358017f-dbd1-4955-9e94-5c66839d5a35)
